components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/ImportCmd.java

```diff
@@ -280,7 +280,7 @@ public void execute() {
             CmdUtils.saveSwaggerDefinitionForMultipleAPIs(projectName, apis,
                     !restVersion.startsWith(CliConstants.REST_API_V1_PREFIX));
         } catch (Exception e) {
-            throw new CLIInternalException("Exception occurred during codeGeneration process");
+            throw new CLIInternalException("Exception occurred during codeGeneration process: " + e.getMessage());
```

Comment:
How about passing `e` to get the whole trace?
```suggestion
            throw new CLIInternalException("Exception occurred during codeGeneration process", e);
```

Reply:
The message is not displaying if we pass the exception